static/app/components/asyncComponent.tsx

```diff
@@ -444,13 +437,12 @@ export default class AsyncComponent<
     );
   }
 
-  shouldRenderLoading() {
-    const {loading, reloading} = this.state;
-    return loading && (!this.shouldReload || !reloading);
+  get shouldRenderLoading() {
+    return this.state.loading && (!this.shouldReload || !this.state.reloading);
   }
 
   renderComponent() {
-    return this.shouldRenderLoading()
+    return this.shouldRenderLoading
```

Comment:
Should this method be called?
```suggestion
    return this.shouldRenderLoading()
```

Reply:
it has the get, so it's not needed

Reply:
yeah I changed it to a get 🤷

Reply:
ah cool, didn't notice that 👍